dataland-frontend/src/components/resources/landing/SampleSection.vue

```diff
@@ -5,10 +5,10 @@
         <div class="grid">
           <div class="col-10 text-left text-gray-100">
             <h2 class="mb-0">Explore Dataland</h2>
```

Comment:
In the figma there is a full stop after "Explore Dataland"

Reply:
yes - and i added a comment there. 
What do you think?

Reply:
Oh ok. I have a small preference for not putting the full stop there, since I interpret it more like a title or action words than a sentence.

Reply:
I leave this comment open until Leo replies and we know what to do

Reply:
Leo replied that he would like to have both full stops.

Reply:
implemented